### PR TITLE
refactor: move benchmark and cli acceptance tests into separate travis tasks

### PR DESCRIPTION
**Pull request by raymondfeng**
1. The benchmark test can be time-consuming and it doesn't involve functional testing. 

2. The CLI acceptance test for `lb4 app` is very slow too as it involves `npm install`.

This PR moves them out of the top-level `npm test` and creates separate Travis tasks to run them as part of CI.

See https://github.com/strongloop/loopback-next/issues/5670

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [x] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "build:site": "./bin/build-docs-site.sh",
     "docs:prepare": "./docs/bin/build-preview-site.sh",
     "docs:start": "cd docs/_preview && bundle exec jekyll serve --no-w --i",
-    "mocha": "node packages/build/bin/run-mocha --lang en_US.UTF-8 --timeout 5000 \"packages/*/dist/__tests__/**/*.js\" \"extensions/*/dist/__tests__/**/*.js\" \"examples/*/dist/__tests__/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\" \"benchmark/dist/__tests__/**/*.js\"",
+    "mocha": "node packages/build/bin/run-mocha --lang en_US.UTF-8 --timeout 5000 \"packages/*/dist/__tests__/**/*.js\" \"extensions/*/dist/__tests__/**/*.js\" \"examples/*/dist/__tests__/**/*.js\" \"packages/cli/test/**/*.js\" \"packages/build/test/*/*.js\"",
```
Comment (agnes512) [inline package.json:72]: Remove `\"packages/cli/test/**/*.js\"` as well?

Comment (raymondfeng) [reply package.json:72]: No, we keep regular CLI tests expect the `lb4 app` acceptance.

**Comment by bajtos** [comment]
I feel it's important to keep running CLI acceptance tests on all platforms we support (Windows, MacOS, Linux) as we were doing before this pull request. @raymondfeng can you please fix that?

**Comment by raymondfeng** [comment]
> I feel it's important to keep running CLI acceptance tests on all platforms we support (Windows, MacOS, Linux) as we were doing before this pull request. @raymondfeng can you please fix that?

Good point. Do we want to make the test conditional on `process.env.CI` flag?

**Comment by bajtos** [comment]
Let's discuss in #5745

**Pull request merged**